### PR TITLE
release: helm: Add the chart as part of the release

### PR DESCRIPTION
**Pull request by fidencio**
So users can simply download the chart and use it accordingly without the need to download the full repo.

### CODE REVIEW

**Comment by fidencio** [comment]
I've tested it here: https://github.com/kata-containers/kata-containers/actions/runs/12187871179

And this is the result: https://github.com/kata-containers/kata-containers/releases/tag/untagged-2b49b4b8debe89e2e191

**Comment by fidencio** [comment]
Once it's approved, I will drop the second commit, and then we can proceed to merge it (and delete the release generated for testing purposes).

**Review by sprt — APPROVED** [review]
LGTM and thanks for testing e2e @fidencio!

File: VERSION
Comment (zvonkok) [inline VERSION:1]: The bump is for testing right?

**Comment by zvonkok** [comment]
So we're only pushing the helm-chart as an artifact?

File: VERSION
Comment (fidencio) [reply VERSION:1]: Yep, the whole second commit is only for testing purposes.

**Comment by fidencio** [comment]
@zvonkok disliked the PR so much that he closed it. :-)

**Review by stevenhorsman — APPROVED** [review]
LGTM (once commit 2 is dropped)

**Comment by fidencio** [comment]
Commit 2 was dropped, release was deleted, and I'm going to merge it with admin powers.

**Pull request merged**
